cluster/addons/storage-crds/csinodeinfo.yaml

```diff
@@ -51,4 +51,11 @@ spec:
                       to use for any in-tree plugins replaced by this driver.
                     pattern: in-tree|csi
```

Comment:
the mapping hasn't been included into k8s/k8s yet. The start of that lives here: https://github.com/kubernetes/kubernetes/pull/70909
it could also use your review